{"uuid": "1abc9378-20c2-4832-86dd-26c40407dc60", "children": ["baa6d5b5-e505-4b36-945d-e682efd0a03c"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\tests\\conftest.py\", line 115, in setup\n    driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()), options=options)\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 45, in __init__\n    super().__init__(\n    ~~~~~~~~~~~~~~~~^\n        browser_name=DesiredCapabilities.CHROME[\"browserName\"],\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n        keep_alive=keep_alive,\n        ^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 55, in __init__\n    self.service.start()\n    ~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 114, in start\n    if self.is_connectable():\n       ~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 131, in is_connectable\n    return utils.is_connectable(self.port)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 101, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"C:\\Python\\Lib\\socket.py\", line 856, in create_connection\n    exceptions.clear()  # raise only the last error\n    ~~~~~~~~~~~~~~~~^^\n"}, "start": 1745242696651, "stop": 1745242699052}], "start": 1745242696651, "stop": 1745242699254}